code/game/area/Space Station 13 areas.dm

```diff
@@ -1382,7 +1382,7 @@ area/space/atmosalert()
 	icon_state = "fitness"
 
 /area/crew_quarters/longue_area

```

Comment:
Yeah, though my choices were:
a) Fix name, but can't port to Polaris, because looks unprofessional since didn't fix path, but can't fix path because it would break their map until updated
b) Fix path and name, but can't port to Polaris, because don't wanna edit their map
c) Fix either path or path+name and just keep it here. Did just name since 1line vs 2line